Mathlib/LinearAlgebra/Matrix/ProjectiveSpecialLinearGroup.lean

```diff
@@ -0,0 +1,36 @@
+/-
+Copyright (c) 2023 Wen Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Wen Yang
+-/
+import Mathlib.LinearAlgebra.Matrix.SpecialLinearGroup
+
+/-!
+# Projective Special Linear Group
+-/
+
+namespace Matrix
+
+universe u v
+
+open Matrix LinearMap
+
+open scoped MatrixGroups
+
+variable (n : Type u) [DecidableEq n] [Fintype n] (R : Type v) [CommRing R]
+
+/-- A projective special linear group is the quotient of a special linear group by its center.-/
+def ProjectiveSpecialLinearGroup :=
```

Comment:
I think this should be an `abbrev`, then you can delete the instances below

Reply:
Amazing!